game/world/objects/npc.cpp

```diff
@@ -2357,7 +2357,13 @@ void Npc::nextAiAction(AiQueue& queue, uint64_t dt) {
       break;
       }
     case AI_Attack:
-      if(currentTarget!=nullptr && weaponState()!=WeaponState::NoWeapon){
+      if(weaponState()==WeaponState::NoWeapon) {
+        // WA for orcgraveyard orcs
+        if(owner.version().game==1 && guild()>GIL_SEPERATOR_ORC)
```

Comment:
would it make more sense to not to check for the orc here?
logic seem to be: if G1 and we are going to attack -> draw some weapon. Also consider `drawWeaponFist` as a case, when no melee is equipped

Reply:
> logic seem to be: if G1 and we are going to attack -> draw some weapon.

I suspected something like this but couldn't prove it but actually there's a simple way. 

Tested vanilla and disabled all script functions that draw a weapon or spell. Command seems to be `drawWeaponMelee`. Every npc without a melee weapon attacks in fist mode. Also active in G2.